modules/kubernetes-addons/aws-efs-csi-driver/locals.tf

```diff
@@ -0,0 +1,87 @@
+locals {
+  name                 = "aws-efs-csi-driver"
+  service_account_name = "efs-csi-sa"
+  namespace            = "kube-system"
+
+  default_helm_config = {
```

Comment:
@luigidifraia this is looking great. One request here...please remove values from `default_helm_config` that are default values in the helm_release resource. We are doing clean up on other add-ons this week to do the same.

Reply:
@kcoleman731 Thank you for taking the time and providing feedback. This should now be addressed in my latest commit.